.github/workflows/release.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
           chainloop version
 
       - name: Initialize Attestation
-        run: chainloop attestation init --contract-revision 2
+        run: chainloop attestation init
```

Comment:
No need to stick to a previous version. I was explicit  in my previous patch because I wanted to make sure the previous workflow worked even after me updating the contract.